src/Platform/PlatformUnix.cpp

```diff
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with Falltergeist.  If not, see <http://www.gnu.org/licenses/>.
  */
+#if defined(__unix__) || defined(__APPLE__) || defined(__linux__)
```

Comment:
@Oppen, because CMake is not a compiler. It doesn't process defines in C++ files